src/modules/api-comment/comments-build.js

```diff
@@ -0,0 +1,26 @@
+import getcomments from './get-comments.js';
+import commentCount from './comment-count.js';
+
+const commentBuild = async (containerClass, url, id, itemID, commentCountContainer) => {
```

Comment:
- [ ] You have done a great job working on the project, all the functionality works as expected. Props on that 👏🏻 . But I noticed that you have 1 modal for both, comments and reservations. However, based on project requirements, you should be having separate modals for comments and reservations: 
<img width="674" alt="Screenshot 2022-03-11 at 13 59 19" src="https://user-images.githubusercontent.com/77038610/157845094-7b9be917-04e0-4a8f-837a-0a94bd0a30a0.png">

- [ ] Please make sure to add the `reservations` button below the `comment` button on the index page, and render a separate modals:

Wireframe layout   | Your layout
------------- | -------------
<img width="169" alt="Screenshot 2022-03-11 at 14 01 44" src="https://user-images.githubusercontent.com/77038610/157845529-8acde3b5-237b-4931-83cf-0c9f639103bc.png"> | <img width="324" alt="Screenshot 2022-03-11 at 14 01 24" src="https://user-images.githubusercontent.com/77038610/157845469-3c0be334-9c96-419d-90f6-c5622b2bc6d2.png">

Reply:
@ShoiraTa Thank you very much for your time and feedback. 
We are a team of 2 only, and I implemented the reservation part as I found it challenging, but it is not part of 2-member team requirements. 

We will be working on modifying the project later on, but for now, the reservation part was extra work I did on the project.

Thank you once again and please, advise if we still need to work on this part.

Reply:
@Elerqsousy  I saw three contributors in this project, it looked like you are a team of 3 😁 sorry for that.  Please request a review again and the project will be approved.